cosmos/views.py

```diff
@@ -412,11 +415,16 @@ def update_door_status(request):
     global door_status
     try:
         request_token = request.GET.get("access_token")
-        status = request.GET.get("status")
+        status = int(request.GET.get("status"))
         pi_token = secret_settings.secrets["TOKENS"]["CR-DOOR"]
 
         if request_token == pi_token:
-            door_status = status
+            if status == 1:
+                open("/tmp/door-open", "w")
```

Comment:
do you not need to close the file as well?

Reply:
You're probably right